### DISCUSSION TITLE
Installing manim on Google Colab

### DISCUSSION THREAD

**Discussion by David-Reinhardt** [ManimCommunity/manim#3771] [Q&A]
Hello!
Does anybody know how to fix the following installation error of Manim on Google Colab?
Thx

> !sudo apt update
…
> !sudo apt install -y ffmpeg libcairo2-dev
…

> !pip install pycairo==1.20.0
Collecting pycairo==1.20.0
  Using cached pycairo-1.20.0.tar.gz (344 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: pycairo
  Building wheel for pycairo (pyproject.toml) ... done
  Created wheel for pycairo: filename=pycairo-1.20.0-cp310-cp310-linux_x86_64.whl size=276368 sha256=32ec1b5d57639c82ba4e6aafbf7f6ebba4915be8972b0f6a67b73e7e53683f16
  Stored in directory: /root/.cache/pip/wheels/e9/92/51/072f9ac6399dc68ec654ff7123073c4f412ed87bc0efea513e
Successfully built pycairo
Installing collected packages: pycairo
Successfully installed pycairo-1.20.0

> !pip install manim
Collecting manim
  Using cached manim-0.18.1-py3-none-any.whl (585 kB)
Requirement already satisfied: Pillow>=9.1 in /usr/local/lib/python3.10/dist-packages (from manim) (9.4.0)
Requirement already satisfied: Pygments>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from manim) (2.16.1)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from manim) (8.1.7)
Collecting cloup>=2.0.0 (from manim)
  Using cached cloup-3.0.5-py2.py3-none-any.whl (54 kB)
Requirement already satisfied: decorator>=4.3.2 in /usr/local/lib/python3.10/dist-packages (from manim) (4.4.2)
Collecting isosurfaces>=0.1.0 (from manim)
  Using cached isosurfaces-0.1.2-py3-none-any.whl (11 kB)
**Collecting manimpango<1.0.0,>=0.5.0 (from manim)
  Using cached ManimPango-0.5.0.tar.gz (4.1 MB)**
  Installing build dependencies ... done
  **error: subprocess-exited-with-error**
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

**Comment by uwezi** [comment]
I just started a fresh new notebook on Colab following the exact instructions on the homepage without any problem...

https://docs.manim.community/en/stable/installation/jupyter.html#google-colaboratory

```
!sudo apt update
!sudo apt install libcairo2-dev ffmpeg \
    texlive texlive-latex-extra texlive-fonts-extra \
    texlive-latex-recommended texlive-science \
    tipa libpango1.0-dev
!pip install manim
!pip install IPython==8.21.0
```

![bild](https://github.com/ManimCommunity/manim/assets/8582807/7f329b7b-0ccc-465b-b0ce-2adb4cf841fb)

**Reply by David-Reinhardt** [reply]
Thanks, worked perfectly. David

**Reply by David-Reinhardt** [reply]
But somehow the video doesn't run

![image](https://github.com/ManimCommunity/manim/assets/124254426/c8d95704-980d-4858-b2bd-d03242f74348)

**Comment by uwezi** [comment]
compare your screenshot with mine...

Currently with Manim 0.18.1 you have to add the line `config.media_embed=True`

**Reply by David-Reinhardt** [reply]
Thanks a lot
